packages/transformers/js/core/src/global_replacer.rs

```diff
@@ -22,171 +22,225 @@ use crate::utils::is_unresolved;
 use crate::utils::SourceLocation;
 use crate::utils::SourceType;
 
+/// Replaces a few node.js constants with literals or require statements.
+/// This duplicates some logic in [`NodeReplacer`]
```

Comment:
Honestly, all of these "globals" are node specific. Could just move it all there, if you're keen for a refactor.

Reply:
It seems there's overlap though; e.g.: there're two implementations of the same thing on these two files.